rules/autocorrect-in-outlook

```diff
@@ -0,0 +1,42 @@
+type: rule
+title: Do you know how to use Autocorrect in Outlook to insert common links you often send?
+uri: /rules/autocorrect-in-outlook
+authors:
+  - title: Ulysses Maclaren
+    url: https://www.ssw.com.au/people/uly
+created: 2024-01-08T13:42:55.753Z
+guid: 7a8b9c0d-3b2e-4f31-af5e-9f9f4a2d9d86
+---
+# Do you know how to use Autocorrect in Outlook to insert common links you often send?
+
+You just received an email from a client asking for your company's Terms and Conditions. Instead of navigating through your files or searching your website to find the link, wouldn't it be more efficient to simply type a shorthand code like `#T` and have Outlook automatically insert the URL for you? This is where the Autocorrect feature in Outlook can become a huge time-saver.
+
+<!--endintro-->
+
+## Why Use Autocorrect for URLs?
+
+- **Efficiency**: It saves time by eliminating the need to find and copy-paste URLs.
+- **Consistency**: It ensures that everyone is using the same URL and reduces the risk of errors.
+  
+## How to Set It Up
+
+1. **Open Outlook** and go to `File` -> `Options`.
+2. Under the `Mail` tab, click on `Spelling and Autocorrect`.
+3. Choose `Autocorrect Options` and add your shorthand and URL.
+
+![Figure: Make sure you use short key combinations that are unlikely to be used normally, but still easy to remember](Outlook-image-uly.png)
```

Comment:
Hi @ae-cher-ssw 

This image was not uploaded... so it would be broken if merged

- [ ] 1. Can you upload the image in this PR? OR do you have another PR?